src/rudderstack.ts

```diff
@@ -74,11 +74,14 @@ export class RudderStack {
 
     init = (RUDDERSTACK_KEY: string) => {
         if (RUDDERSTACK_KEY) {
+            const _define = window.define
```

Comment:
@shayan-deriv my concern is that we are trying to fix a problem only for Outsystems in a package that is getting used in different Frameworks and not only in Outsystems.
Will this cause any issue?

Reply:
good point. then how about passing a flag for having this logic(which is false by default) ? 🤔

Reply:
what kind of flag?

Reply:
like this:

```
init = (RUDDERSTACK_KEY: string, disableAMD: boolean = false) => {
        if (RUDDERSTACK_KEY) {
            let _define: any;
            if (disableAMD) {
                _define = window.define;
                window.define = undefined;
            }

            this.setCookieIfNotExists();
            this.analytics.load(RUDDERSTACK_KEY, 'https://deriv-dataplane.rudderstack.com', {
                externalAnonymousIdCookieName: this.rudderstack_anonymous_cookie_key,
            });
            this.analytics.ready(() => {
                if (disableAMD) {
                    window.define = _define;
                }
                this.has_initialized = true;
                this.has_identified = !!(this.getUserId() || this.getAnonymousId());
                this.handleCachedEvents();
            });
        }
    }
    ```